diagnostic.md

```diff
@@ -6,70 +6,76 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+get '/books', to: 'books#index'
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+config/routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+rake db:migrate generates a migration file. In the file, Rails creates a class that inherits from ActiveRecord:: Migration. In other words, this is Rails way of telling the database what to do. The class will have generated methods specified by what you command.
```

Comment:
`bundle exec rake db:migrate` executes all of the database migration files in `db/migrate` and generates a new database schema file, `db/schema.rb`.